app/assets/stylesheets/container_topology.css

```diff
@@ -53,3 +53,12 @@
 .kube-topology g.ContainerManager text.glyph {
     font-size: 38px;
 }
+
+.topology-search {
+    width: auto;
+    display: inline;
+}
+
+.topology-checkbox {
+    padding-left: 26px;
+}
```

Comment:
I think the above 2 changes belong [here](https://github.com/ManageIQ/manageiq/blob/master/app/assets/stylesheets/topology.css)